pkg/serverless/aws/arn.go

```diff
@@ -172,103 +141,3 @@ func RestoreCurrentStateFromFile() error {
 	SetRequestID(restoredState.CurrentReqID)
 	return nil
 }
-
-// FetchFunctionARNFromEnv reconstructs the function arn from what's available
-// in the environment.
-func FetchFunctionARNFromEnv(accountID string) (string, error) {
```

Comment:
need to double check the behaviour in us-gov but my understanding is that we can get rid of this logic since `InvokedFunctionArn` will be already well formatted (and prefixed with `aws-us-gov`)